src/components/mixins/entity_list.js

```diff
@@ -428,6 +428,7 @@ export const entityListMixin = {
     validationColumnsIsInDepartmentFilter(columnId) {
       return (
         this.departmentFilter.length === 0 ||
+        this.taskTypeMap.get(columnId).department_id === null ||
```

Comment:
`this.taskTypeMap.get(columnId)` will never be empty ?

else add an optional chaining:
```suggestion
        this.taskTypeMap.get(columnId)?.department_id === null ||
```